### ISSUE TITLE
update link

### ISSUE THREAD

**Issue by saracope** [GSA/digitalgov-usability#35]
https://www.usability.gov/how-to-and-tools/resources/templates/system-usability-scale-sus.html

note from email:

>  the link to usability.net on this page, http://www.usabilitynet.org/trump/documents/Suschapt.doc, is being blocked by our Cisco Umbrella Service. It appears that usabilitynet.org just redirects to usability.net which is not blocked. Is there a more direct link that you can provide to the document?

**Comment by ToniBonittoGSA** [comment]
was fixed via another Issue
